docs/bpmn-support.adoc

```diff
@@ -160,16 +160,16 @@ The default rendering uses `white` as fill color and `black` as stroke color.
 |Comments
 
 |sequence flow
-|icon:flask[]
-|Subject to change: arrow shape, color and position endpoint
+|icon:check-circle-o[]
+|Subject to change: arrow size/form and position endpoint
 
 |default sequence flow
-|icon:flask[]
-|Subject to change: arrow shape, color and position endpoint
+|icon:check-circle-o[]
+|Subject to change: arrow size/form and position endpoint
 
 |conditional sequence flow
-|icon:flask[]
-|Subject to change: arrow shape, color and position endpoint
+|icon:check-circle-o[]
+|Subject to change: arrow size/form and position endpoint
```

Comment:
:warning: missing ref to draw.io marker

Reply:
It's normal, beacause I didn't search on draw.io how they do to make this marker 😉 
I just searched on the documentation of MxGraph.

Reply:
I have been tricked by the commits from the default seq flow render which use marker inspired from draw.io.